LayoutTests/imported/w3c/web-platform-tests/svg/coordinate-systems/out-of-flow-svg-root.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<link rel="help" href="https://www.w3.org/TR/SVG2/coords.html">
```

Comment:
If it is not too much trouble, it would be good to add specification reference, if any.